config/cmake/config/vxl_platform_tests.cxx

```diff
@@ -270,13 +281,19 @@ int main() { return f(&A::x); }
 
 #ifdef VCL_STATIC_CONST_INIT_FLOAT
 
+// If __cplusplus is C++11 (greater than equal to 201103L)
+// then constexpr floating point initialization is supported.
+#if  __cplusplus >= 201103L
 class A {
  public:
-  static const float x = 27.0f;
-  static const double y = 27.0;
+  static VCL_CONSTEXPR float x = 27.0f;
+  static VCL_CONSTEXPR double y = 27.0;
 };
-
 int main() { return A::x == 27.0f && A::y == 27.0 ? 0 : 1; }
+#else
+  #error "VCL_STATIC_CONST_INIT_FLOAT only supported on gcc compilers (and c++0x).
```

Comment:
This is missing the trailing `"` which causes the check to fail for me.  Then we end up with specializations missing `template <>`.